components/odh-notebook-controller/controllers/notebook_runtime_test.go

```diff
@@ -7,10 +7,10 @@ import (
 	nbv1 "github.com/kubeflow/kubeflow/components/notebook-controller/api/v1"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	imagev1 "github.com/openshift/api/image/v1"
 	corev1 "k8s.io/api/core/v1"
 	apierrs "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
could you group the imports according to meaning?

leave blank line between the `. ` imports, then do the schema imports like `nbv1`, `imagev1`, and then another blank line and then the other stuff?

Reply:
in go it usually remains how you leave it; afaik there has not been any strong fixation on tooling for import ordering in go